tools/defaults.bzl

```diff
@@ -1,9 +1,10 @@
 """Re-export of some bazel rules with repository-wide defaults."""
+
 load("@build_bazel_rules_typescript//:defs.bzl", _ts_library = "ts_library")
 
 DEFAULT_TSCONFIG = "//:tsconfig.json"
 
 def ts_library(tsconfig = None, **kwargs):
-  if not tsconfig:
-    tsconfig = DEFAULT_TSCONFIG
-  _ts_library(tsconfig = tsconfig, **kwargs)
+    if not tsconfig:
```

Comment:
actually you can remove this now, `ts_library#tsconfig` defaults to `//:tsconfig.json` as of release 0.15 or so

Reply:
Done